pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/redigerbar/InnhentingInformasjonFraBrukerDto.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.pensjon.brev.api.model.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.api.model.maler.EmptyBrevdata
+import no.nav.pensjon.brev.api.model.maler.RedigerbarBrevdata
+
+@Suppress("unused")
+data class InnhentingInformasjonFraBrukerDto(
+    override val saksbehandlerValg: SaksbehandlerValg,
+    override val pesysData: EmptyBrevdata
+) : RedigerbarBrevdata<InnhentingInformasjonFraBrukerDto.SaksbehandlerValg, EmptyBrevdata> {
+    data class SaksbehandlerValg(
+        val bosattIEoesLandSedErEoesBlanketter: Boolean, //Bosatt i EØS-land. SED-er/EØS-blanketter
```

Comment:
Kom på at skribenten også forsøker å beholde casing for akronym. Så `harAFPOgUFO` blir til `Har AFP og UFO`. Men denne koden er ikke veldig forutsigbar, og er ganske spesifikk. Jeg vil anbefale å teste navnet [her](https://github.com/navikt/pensjonsbrev/blob/9d25a2419bb9c2589bf20fe6a3db71db745dcd6d/skribenten-web/frontend/test/modules/ModelEditor/components/utils.test.ts).

Reply:
Frister mer å begynne på displayName for saksbehandlerValg

Reply:
Dine blir:
```
bosattIEoesLandSedErEoesBlanketter: Bosatt eøs land sed er eøs blanketter
inntektsopplysninger: Inntektsopplysninger
bankOpplysninger: Bank opplysninger
amerikanskSocialSecurityNumber: Amerikansk social security number
registreringAvSivilstand: Registrering av sivilstand
eps60aarOgInntektUnder1G: Eps60år og inntekt under1
eps62aarOgInntektUnder1GBoddArbeidUtland: Eps62år og inntekt under1 bodd arbeid utland
epsInntektUnder2G: Eps inntekt under2
forsoergerEpsBosattIUtlandet: Forsørger eps bosatt utlandet
tidspunktForUttak: Tidspunkt for uttak
manglendeOpptjening: Manglende opptjening
boOgArbeidsperioder: Bo og arbeidsperioder
bosattEoesLandSedEoesBlanketter: Bosatt eøs land sed eøs blanketter
```

Reply:
Samboer blir sambør

Reply:
epsInntektUnder2G: Eps inntekt under2. G forsvinner.

Reply:
Etter noen små rettinger blir det: